department-of-veterans-affairs/va.gov-cms
# VACMS-5282: Replace button on user login form with get help link.

## Description
See #5282

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/118033937-612f6000-b337-11eb-9839-08b21066babb.png)

## QA steps
- As an anonymous user, go to '/' and visually verify that the `Get help` button appears, and clicking it takes you to `/help`

### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
